ts/Series/CenteredUtilities.ts

```diff
@@ -78,7 +79,8 @@ namespace CenteredUtilities {
             plotWidth = chart.plotWidth - 2 * slicingRoom,
             plotHeight = chart.plotHeight - 2 * slicingRoom,
             centerOption: Array<(number|string|null)> = options.center as any,
-            smallestSize = Math.min(plotWidth, plotHeight);
+            smallestSize = Math.min(plotWidth, plotHeight),
+            thickness = options.thickness || 0;
```

Comment:
You don't need `|| 0` here because you are checking `isNumber(thickness)` below. If TypeScript will mess with you about it, send him to me and we'll explain this and that...! (if you know what I mean)

...
(simply check `isNumber(thickness)` **first** in your `if` statement)